stanford/roles/cluster/defaults/main.yml

```diff
@@ -257,7 +257,7 @@ CLUSTER_SECURITY_GROUP_RULES:
       cidr_ip: '0.0.0.0/0'
 CLUSTER_ELBS:
   analytics-api:
-    - name: "{{ COMMON_DEPLOYMENT }}-elb-analytics-api-public"
+    - name: "{{ COMMON_DEPLOYMENT }}-elb-analytics-api-publi"
```

Comment:
Yeah, this is what I warned about when we started on these..

I'm really not a fan of this truncation;
it breaks one of my regular auditing tools.
I sometimes search for all public ELBs by searching for `public`
and now my queries won't catch this.

We should come up w/ another shorter, non-hyphenated cluster name;
`dataapi`?